Sources/StreamChatUI/ChatMessageList/ChatMessageListIndicatorOverlayView.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// An `UIView` subclass indicating that user or multiple users are currently typing.
+public typealias ChatMessageListTypingIndicatorView = _ChatMessageListTypingIndicatorView<NoExtraData>
+
+/// An `UIView` subclass indicating that user or multiple users are currently typing.
+open class _ChatMessageListTypingIndicatorView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
```

Comment:
No more crazy long names please 😅  `TypingIndicatorView` is enough for our needs